com.airtame.Client.appdata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.airtame.Client</id>
```

Comment:
Missing `<launchable>`

Reply:
Fixed

Reply:
> Missing `<launchable>`

Does `<launchable>` work with `extra-data` apps? From [this commit](https://github.com/flathub/com.spotify.Client/commit/e1344686ec49eb47efc94ac1471649d4ba1cd139) I get the feeling it dosen't (?).